src/fetch/ipfs.py

```diff
@@ -49,3 +56,55 @@ def get_content(self, max_retries: int = 3) -> Optional[Any]:
             except requests.exceptions.ReadTimeout:
                 attempts += 1
         return None
+
+    @classmethod
+    async def fetch_many(
+        cls, missing_rows: list[dict[str, str]], max_retries: int = 3
+    ) -> tuple[list[FoundContent], list[NotFoundContent]]:
+        """Async AppData Fetching"""
+        found, not_found = [], []
+        async with aiohttp.ClientSession() as session:
```

Comment:
This thing @MartinquaXD - I believe to be equivalent to `asycio.gather()` as is an alternate (less easy to work with variant).